Makefile

```diff
@@ -3,8 +3,9 @@ MAKEFLAGS+=-r
 
 config=debug
 defines=standard
+cxxstd=default
```

Comment:
Can you default to c++11? In general if pugixml compiles in c++11 it should compile in other standards as well; AppVeyor build generally checks that it does not use C++11 feature unless they are present.